cluster-autoscaler/cloudprovider/node_group_discovery_options.go

```diff
@@ -186,8 +186,8 @@ func parseASGAutoDiscoverySpec(spec string) (ASGAutoDiscoveryConfig, error) {
 		return cfg, fmt.Errorf("Unsupported discoverer specified: %s", discoverer)
 	}
 	param := tokens[1]
-	kv := strings.Split(param, "=")
-	if len(kv) != 2 {
+	kv := strings.SplitN(param, "=", 2)
+	if len(kv) < 2 {
```

Comment:
why are you ok with len(kv) > 2 ?

Reply:
You're right, this should stay `!= 2`. Fixed it!